certbot/tests/util.py

```diff
@@ -169,6 +170,26 @@ def patch_get_utility(target='zope.component.getUtility'):
     return mock.patch(target, new_callable=_create_get_utility_mock)
 
 
+def patch_get_utility_with_stdout(target='zope.component.getUtility',
+                                  stdout=six.StringIO()):
```

Comment:
I liked your previous approach of using `None` here better. The problem with this is this is the default is set to a specific `StringIO` object when the function is defined so different calls to the function using this default will get the same object causing interactions between tests.

Reply:
Updated it to default to `None`.